package.json

```diff
@@ -102,6 +102,7 @@
     "@wordpress/i18n": "3.5.0",
     "@wordpress/jest-preset-default": "4.0.0",
     "@wordpress/scripts": "12.4.0",
+    "@wordpress/url": "2.22.0",
```

Comment:
I see references to `@wordpress/url` but you just added it here. How did it work before?